libs/analysis/bbhnet/analysis/distributions/cluster.py

```diff
@@ -99,3 +99,11 @@ def nb(self, threshold: Union[float, np.ndarray]):
         else:
             nb = np.sum(self.events >= threshold)
         return np.array(nb)
+
+    def copy(self):
```

Comment:
Is there a reason to add this method here and to the `Foreground` object rather than using Python's built-in `copy.deepcopy` function?

Reply:
Nope. Wasn't familiar with it. Good to know, will adjust to the built in version.